apps/extension/client/README.md

```diff
@@ -4,6 +4,7 @@ XState VSCode enhances the XState development experience by providing VSCode use
 
 1. [Install via the Visual Studio Code Marketplace →](https://marketplace.visualstudio.com/items?itemName=statelyai.stately-vscode)
 2. Once installed, you can run `XState: Open Visual Editor` from the command palette to open any machine at your cursor's location.
+3. If you have code lens enabled, you'll also see 'Open Visual Editor' floating above each `createMachine` call.
```

Comment:
```suggestion
3. If you have code lens enabled (this can be enabled using [`editor.codeLens`](https://code.visualstudio.com/docs/getstarted/settings#_default-settings) setting), you'll also see 'Open Visual Editor' floating above each `createMachine` call.
```